uniffi_bindgen/src/interface/visit_mut.rs

```diff
@@ -131,6 +131,14 @@ impl ComponentInterface {
                 }
             }
         }
+
+        self.errors = self.errors
+            .drain()
+            .map(|mut name| {
+                visitor.visit_error_name(&mut name);
+                name
+            })
+            .collect()
```

Comment:
This is the basic issue, this is a somewhat hacky field that stores error type names when we see them in "[Throws=]" UDL attributes.  I think that proc-macros might have handled this correctly, but I'm not sure.

Reply:
I like `drain` in combination with `collect`. I think I could have used something similar here as well!